ztp/ztp-policy-generator/testPolicyGenTemplate/group-du-sno-ranGen.yaml

```diff
@@ -49,7 +49,7 @@ spec:
           machineconfiguration.openshift.io/role: master
     - fileName: MachineConfigDisableChronyd.yaml
       policyName: "mc-chronyd-policy"
-    - fileName: PtpConfigSlave.yaml
+    - fileName: PtpConfigSlave.yaml  # Change to PtpConfigSlaveCvl.yaml for ColumbiaVille NIC
```

Comment:
This change needs to happen in the "real" DU profile :)

Reply:
You mean in this one below?
https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/resource-hook-example/policygentemplates/group-du-sno-ranGen.yaml

Reply:
Exactly.

Reply:
Done